.github/workflows/ci.yml

```diff
@@ -475,6 +478,8 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: .
+          build-contexts: local-stash=docker-image://localhost:5000/local-stash:latest
+          load: true
```

Comment:
We don't need it anymore - but let's delete it in the next PR :)

Reply:
same as line 492&&493